### DISCUSSION TITLE
python3 -m build . not working

### DISCUSSION THREAD

**Discussion by seohyeonlee2020** [USEPA/pyaqsapi#9] [Q&A]
Hello! I'm trying to install pyaqsapi but I'm repeatedly getting this error message:
ERROR Source . does not appear to be a Python project: no pyproject.toml or setup.py

What should I do? Thank you for reading!

**Comment by mccroweyclinton-EPA** [comment]
make sure that you are in the project's root directory when attempting to run the build command. There is also a sub directory with the same name. You may have accedently tried to build the package in the pyaqsapi sub directory. The root directory has both files availble.

**Comment by mccroweyclinton-EPA** [comment]
I have updated the README to help clarify that building should be done in the projects root directory.  Hopefully that will make things clearer for others wanting to build the package. See [a39ff6c](https://github.com/USEPA/pyaqsapi/commit/a39ff6c9b6f69b25feaae52b241d0c1105b54401).
